pkgs/development/python-modules/elpy/default.nix

```diff
@@ -1,32 +1,43 @@
 { stdenv
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , rope
 , flake8
 , autopep8
 , jedi
 , importmagic
-, isPy27
+, black
+, mock
+, nose
+, yapf
+, isPy3k
 }:
 
 buildPythonPackage rec {
   pname = "elpy";
-  version = "1.28.0";
+  version = "1.29.1";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "0lx6bf6ajx6wmnns03gva5sh1mmmxahjaqrn735cgwn6j4ikyqfs";
+  src = fetchFromGitHub {
+    owner = "jorgenschaefer";
+    repo = "elpy";
+    rev = "1.29.1";
```

Comment:
Do you mind using ```${version}```? Bots are using it to attempt automatically updating the packages.